### PR TITLE
Model card additions

### PR DESCRIPTION
**Pull request by EziOzoani**
We want users to be able to more easily find the [model card creator tool](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool) when they’re making cards, so I’m linking the space within the model card document for when users are reading the model cards hub doc.

### CODE REVIEW

File: docs/hub/model-cards.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Model Cards
 
+  ### [<Tip> New! Try our experimental Modal Card Creator App</Tip>](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool)
```
Comment (julien-c) [inline docs/hub/model-cards.md:3]: <Tip></Tip> must wrap outside of the inner content i think

File: docs/hub/model-cards.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 # Model Cards
 
+  ### <Tip>[ New! Try our experimental Modal Card Creator App](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool)</Tip>
```
Comment (osanseviero) [inline docs/hub/model-cards.md:3]:
```suggestion
<Tip>
[New! Try our experimental Model Card Creator App](https://huggingface.co/spaces/huggingface/Model_Cards_Writing_Tool)
</Tip>
```

**Comment by osanseviero** [comment]
This is how it would look like https://moon-ci-docs.huggingface.co/docs/hub/pr_452/en/model-cards

cc @meg-huggingface @EziOzoani. WDYT?

File: docs/hub/model-cards.md (hunk 2 above)
Comment (EziOzoani) [reply docs/hub/model-cards.md]: I can't seem to comment on your comment below but it looks good to me 
thanks, @osanseviero and @julien-c for your inputs 🤗

**Comment by meg-huggingface** [comment]
I actually can't see anything at this link: https://moon-ci-docs.huggingface.co/docs/hub/pr_452/en/model-cards

But just judging from this PR, seems okay to me!

**Review by osanseviero — APPROVED** [review]
Let's merge it!

**Pull request merged**